docs/src/main/sphinx/connector/system.md

```diff
@@ -81,8 +81,6 @@ The materialized views table contains the following information about all
   - Freshness of data in the storage table. Queries on the materialized view
     access the storage table if not `STALE`, otherwise the `definition` is used
     to access the underlying data in the source tables.
-* - `owner`
```

Comment:
`last_fresh_time` column is missing.
https://github.com/trinodb/trino/blob/0edd4b429f4f6d3fbba83bbe0545a46dfa03d6a4/core/trino-main/src/main/java/io/trino/connector/system/MaterializedViewSystemTable.java#L60-L72

Reply:
Thanks for taking a look, @ebyhr. I have added the `last_fresh_time` column and a description.